src/commcare_cloud/ansible/roles/postgresql/tasks/main.yml

```diff
@@ -71,15 +71,6 @@
 - name: assert replication slot config correct
   assert: { that: "replication_slot is not defined or replication_slot in hostvars[hot_standby_master]['postgresql_replication_slots']"}
 
-- name: Create wal archive directory
```

Comment:
should this be actively removed from servers where it still exists?

Reply:
I suppose so. I'll update to remove it.

Reply:
d970298